src/cephadm/cephadm

```diff
@@ -2291,6 +2295,24 @@ def command_adopt():
             call_throws(['umount', data_dir_src])
         os.rmdir(data_dir_src)
 
+        logger.info('Chowning content...')
+        call_throws(['chown', '-c', '-R', '%d.%d' % (uid, gid), data_dir_dst])
+
+        if daemon_type == 'mon':
+            # rename *.ldb -> *.sst, in case they are coming from ubuntu
+            store = os.path.join(data_dir_dst, 'store.db')
+            num_renamed = 0
+            if os.path.exists(store):
+                for f in os.listdir(store):
+                    if f.endswith('.ldb'):
+                        n = f.replace('.ldb', '.sst')
+                        fp = os.path.join(store, f)
+                        np = os.path.join(store, n)
+                        logger.debug('Renaming %s -> %s' % (np, fp))
+                        os.rename(fp, np)
```

Comment:
Which direction `.ldb -> .sst` or `.sst -> .ldb`?

Feels like the log line doesn't match the rename...